### DISCUSSION TITLE
Connect copilot with VS Code

### DISCUSSION THREAD

**Discussion by Sonnig92** [community/community#30501] [Copilot Conversations]
I have 2 months trial to use Copilot. but when i login from vs code the github site always open the billing page and i cant connect copilot with vs code. What should i do. Thanks!

**Comment by lonasozo** [comment]
Same error of this? #30494

You could try to install the nightly version

**Reply by Sonnig92** [reply]
OMG. Thank you so much. It is workingggg

**Reply by Beef-Boy** [reply]
I had the same problem, it worked with nightly aswell
